crates/ruff/src/rules/flake8_bandit/rules/try_except_continue.rs

```diff
@@ -0,0 +1,42 @@
+use ruff_macros::{define_violation, derive_message_formats};
+use rustpython_parser::ast::{Expr, Stmt, StmtKind};
+
+use crate::ast::types::Range;
+use crate::checkers::ast::Checker;
+use crate::registry::Diagnostic;
+use crate::violation::Violation;
+
+define_violation!(
+    pub struct TryExceptContinue;
```

Comment:
I'd love to get into the habit of adding docs to new rules now that we support them :) Check out `UnconventionalImportAlias` as an example. We should make it a requirement for new checks.